reader/reader/core/reader_service.py

```diff
@@ -12,11 +19,31 @@
 
 @service_as_factory
 class ReaderService:
-    def get(self, request: GetRequest):
-        return request.fqid
+
+    database: ReadDatabase
+
+    def get(self, request: GetRequest) -> Dict[str, Any]:
+        with self.database.get_context():
+            models = self.database.get_models_filtered(
+                [request.fqid], request.get_deleted_models
+            )
+            if request.fqid not in models:
+                raise ModelDoesNotExist(request.fqid)
+            return self.apply_mapped_fields(models[request.fqid], request.mapped_fields)
 
     def get_many(self, request: GetManyRequest):
-        pass
+        result_set = []
+        with self.database.get_context():
+            for part in request.requests:
+                fqids = [build_fqid(part.collection, id) for id in part.ids]
+                models = self.database.get_models_filtered(
+                    fqids, request.get_deleted_models
+                )
+
+                result_set += self.apply_mapped_fields_multi(
+                    models.values(), part.mapped_fields + request.mapped_fields
+                )
+        return jsonify(result_set)
```

Comment:
Maybe you should return the list here and convert the data to the desired outputformat in the frontend.